setup.cfg

```diff
@@ -18,6 +18,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+    bottleneck>=1.3.4
```

Comment:
why do we need this? I don't think we're using that anywhere?

Reply:
pandas was complaining that this version of bottleneck was required. Hence this addition.

Reply:
Mhm, this seems to be only an optional dependency from pandas:
https://github.com/pandas-dev/pandas/blob/f5d754d4fcaefff9ff08167a55426f3afe88b175/pyproject.toml#L67

did you try this in a clean environment? I works fine for me without it...

Reply:
Ha, yes, you are right ... this can indeed be removed then!